topology/src/test/java/io/camunda/zeebe/topology/PersistedClusterTopologyTest.java

```diff
@@ -9,86 +9,77 @@
 
 import static org.junit.jupiter.api.Assertions.*;
 
-import io.atomix.cluster.MemberId;
 import io.camunda.zeebe.topology.serializer.ProtoBufSerializer;
-import io.camunda.zeebe.topology.state.ClusterChangePlan;
 import io.camunda.zeebe.topology.state.ClusterTopology;
-import io.camunda.zeebe.topology.state.CompletedChange;
-import io.camunda.zeebe.topology.state.MemberState;
-import io.camunda.zeebe.topology.state.TopologyChangeOperation;
-import io.camunda.zeebe.util.ReflectUtil;
 import java.io.IOException;
 import java.nio.file.Files;
-import net.jqwik.api.Arbitraries;
-import net.jqwik.api.Arbitrary;
-import net.jqwik.api.Combinators;
-import net.jqwik.api.ForAll;
-import net.jqwik.api.Property;
-import net.jqwik.api.Provide;
-import net.jqwik.api.domains.Domain;
-import net.jqwik.api.domains.DomainContext;
-import net.jqwik.api.domains.DomainContextBase;
+import java.nio.file.StandardOpenOption;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.Test;
 
 final class PersistedClusterTopologyTest {
```

Comment:
:upside_down_face:  I would add a test with some bit flip, i.e. write a new "correct" topology but don't update the checksum. so it's testing that everything is parse-able, but doesn't match. Soft suggestion because tbh `shouldDetectFileWithMoreDataThanExpected` already asserts the failure is checksum related, so the important invariant is already tested